src/hooks/useNormalizeEvolutionChain.js

```diff
@@ -0,0 +1,11 @@
+import { useMemo } from 'react';
+import { normalizeEvolutionChain } from '../utils';
+
+// Hook to normalize an evolution chain into an array.
+const useNormalizeEvolutionChain = ( { chain, id } ) => {
+	return useMemo( () => {
+		return { evolutionChain: chain ? normalizeEvolutionChain( chain ) : [] };
+	}, [ id ] );
```

Comment:
Are you sure you don't want to return an object instead of the entire useMemo with the object inside of it? - just for readability...